doc/nrf/releases_and_maturity/migration/migration_guide_cs3_to_2_6_99.rst

```diff
@@ -0,0 +1,976 @@
+
+   * For the :ref:`crypto_persistent_key` sample:
+
+     * The Kconfig option ``CONFIG_PSA_NATIVE_ITS`` is replaced by the Kconfig option :kconfig:option:`CONFIG_TRUSTED_STORAGE`, which enables the new :ref:`trusted_storage_readme` library.
+       The :ref:`trusted_storage_readme` library provides the PSA Internal Trusted Storage (ITS) API for build targets without TF-M.
+       It is not backward compatible with the previous PSA ITS implementation.
+       Migrating from the PSA ITS implementation, enabled by the ``CONFIG_PSA_NATIVE_ITS`` option, to the new :ref:`trusted_storage_readme` library requires manual data migration.
+
+   * For :ref:`lib_wifi_credentials` library and Wi-Fi samples:
+
+     * ``CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET`` has been removed because it was specific to the previous solution that used PSA Protected Storage instead of PSA Internal Trusted Storage (ITS).
+       Use :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_OFFSET` to specify the key offset for PSA ITS.
+       Be aware that Wi-Fi credentials stored in Protected Storage will not appear in ITS when switching.
+       To avoid re-provisioning Wi-Fi credentials, manually read out the old credentials from Protected Storage in the previously used UID and store to ITS.
+
+zcbor
+=====
+
+.. toggle::
+
+   * If you have zcbor-generated code that relies on the zcbor libraries through Zephyr, you must regenerate the files using zcbor 0.8.1.
+     Note that the names of generated types and members has been overhauled, so the code using the generated code must likely be changed.
+
+     For example:
+
+      * Leading single underscores and all double underscores are largely gone.
+      * Names sometimes gain suffixes like ``_m`` or ``_l`` for disambiguation.
+      * All enum (choice) names have now gained a ``_c`` suffix, so the enum name no longer matches the corresponding member name exactly (because this broke C++ namespace rules).
+
+    * The function :c:func:`zcbor_new_state`, :c:func:`zcbor_new_decode_state` and the macro :c:macro:`ZCBOR_STATE_D` have gained new parameters related to decoding of unordered maps.
+      Unless you are using that new functionality, these can all be set to NULL or 0.
+    * The functions :c:func:`zcbor_bstr_put_term` and :c:func:`zcbor_tstr_put_term` have gained a new parameter ``maxlen``, referring to the maximum length of the parameter ``str``.
+      This parameter is passed directly to :c:func:`strnlen` under the hood.
+    * The function :c:func:`zcbor_tag_encode` has been renamed to :c:func:`zcbor_tag_put`.
+    * Printing has been changed significantly, for example, :c:func:`zcbor_print` is now called :c:func:`zcbor_log`, and :c:func:`zcbor_trace` with no parameters is gone, and in its place are :c:func:`zcbor_trace_file` and :c:func:`zcbor_trace`, both of which take a ``state`` parameter.
+
+MCUboot
```

Comment:
MCUboot is not supported for nRF54H, and the mcuboot porting for nRF54L15 was not ready for customer sampling.
Yeah -maybe someone did something on their own - the note bellow is in general valid.